localstack_services.sh

```diff
@@ -27,7 +27,7 @@ tfswitch 1.9.2
 
 basedir=$(pwd)
 
-if ! curl https://localhost.localstack.cloud:4566/_localstack/health > /dev/null 2>&1; then
+if ! curl http://localhost.localstack.cloud:4566/_localstack/health > /dev/null 2>&1; then
```

Comment:
switched to non-tls due to an ongoing incident https://localstack.statuspage.io/